__std__/nix/automation/hydraJobs.nix

```diff
@@ -25,37 +31,33 @@ let
 
   windows-plutus-jobs = make-haskell-jobs plutus-project.projectCross.mingwW64;
 
-  other-jobs = {
-    plutus.shells = inputs.cells.plutus.devshells;
-    doc.shells = inputs.cells.doc.devshells;
-    doc.packages = inputs.cells.doc.packages;
-    doc.scripts = inputs.cells.doc.scripts;
-    toolchain.packages = inputs.cells.toolchain.packages;
-    toolchain.scripts = inputs.cells.toolchain.scripts;
-  };
+  other-jobs =
+    inputs.cells.plutus.devshells //
+    inputs.cells.doc.devshells //
+    inputs.cells.doc.packages //
+    inputs.cells.doc.scripts //
+    inputs.cells.toolchain.packages;
 
   jobs =
-    if system == "x86_64-linux" then {
-      plutus.windows = windows-plutus-jobs;
-      plutus.native = native-plutus-jobs;
-      other = other-jobs;
-    } else if system == "x86_64-darwin" then {
-      plutus.native = native-plutus-jobs;
-      other = other-jobs;
-    } else { };
+    if system == "x86_64-linux" then
+      { native = native-plutus-jobs; windows = windows-plutus-jobs; } // other-jobs
+    else if system == "x86_64-darwin" then
+      { native = native-plutus-jobs; } // other-jobs
+    else
+      { };
 
   # Hydra doesn't like these attributes hanging around in "jobsets": it thinks they're jobs!
-  filtered-jobs = lib.filterAttrsRecursive (n: _: n != "recurseForDerivations") jobs;
+  # filtered-jobs = lib.filterAttrsRecursive (n: _: n != "recurseForDerivations") jobs;
```

Comment:
just an experiment, we probably do need it

Reply:
Turns out hydra does complain, but hydra is going away anyway... So I just added a TODO